resources/keb/values.yaml

```diff
@@ -330,6 +330,8 @@ oidc:
     orchestrations: orchestrationsAdmin
     viewer: runtimeViewer
 
+oidcs: []
```

Comment:
instead, I would just have a new issuers list with empty default under the existing oidc config:

```suggestion
oidc:
  issuers: []
```